src/scene/data.ts

```diff
@@ -11,6 +11,9 @@ import nanoid from "nanoid";
 
 const LOCAL_STORAGE_KEY = "excalidraw";
 const LOCAL_STORAGE_KEY_STATE = "excalidraw-state";
+const JSONSTOREIO_SHORTLINK_ENDPOINT = new URL(
+  "https://www.jsonstore.io/7535731590bfdc991d38b97d0fec2b722f5701c15118e0b1896968c7fe9f07a9"
```

Comment:
What is this long id?

Reply:
AFAIK that's the namespace we get to make requests against. It should be unique to us.

Reply:
Yep, it just generates a unique namespace for all our links